clang/include/clang/CConv/CheckedRegions.h

```diff
@@ -79,6 +80,7 @@ class CheckedRegionFinder : public RecursiveASTVisitor<CheckedRegionFinder>
     bool isUncheckedPtrAcc(QualType Qt, std::set<std::string> &Seen);
     bool isUncheckedStruct(QualType Qt, std::set<std::string> &Seen);
     bool isFunctionBody(CompoundStmt *S);
+    bool isWild(std::set<ConstraintVariable*>);
```

Comment:
I think this would be better as a reference.

Reply:
Done.